ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -562,7 +562,10 @@ def get_osd_count():
 
     """
     sc = get_storage_cluster()
-    return int(sc.get().get("items")[0]["spec"]["storageDeviceSets"][0]["count"]) * int(
+    sc_data = sc.get().get("items")[0]
+    if sc_data["spec"].get("externalStorage", {}).get("enable"):
+        return 0
```

Comment:
update the docstring to state returns 0 for external cluster?

Reply:
This function is retuning number of OSD pods which in the case of external cluster is 0.  I can add it to docstring.